lona/client/dom-updater.js

```diff
@@ -439,15 +441,20 @@ Lona.LonaDomUpdater = function(lona_context, lona_window) {
             // SET
             if(operation == protocol.OPERATION.SET) {
                 var name = data[0];
+                var value = data[1];
 
                 // properties
                 if(property_names.indexOf(name) > -1) {
-                    node[data[0]] = data[1];
+                    if(name != 'value') {
+                        value = true;
+                    };
+
+                    node[name] = value;
 
                 // attributes
                 } else {
-                    node.setAttribute(data[0], data[1]);
-                }
+                    node.setAttribute(name, data[1]);
+                };
 
             // RESET
```

Comment:
Don't we need to do the same check on reset?

Reply:
Ack! I will fix that.